servers/jax-rs/src/test/java/org/projectnessie/jaxrs/AbstractTestRest.java

```diff
@@ -920,6 +920,7 @@ void invalidBranchNames(String invalidBranchName, String validHash) {
     Contents cts = IcebergTable.of("moo");
     MultiGetContentsRequest mgReq = MultiGetContentsRequest.of(key);
     Tag tag = Tag.of("valid", validHash);
+    String ref = "1234567890123456";
```

Comment:
Where's `ref` being used?

Reply:
it's a leftover that needs to be removed